Dockerfile

```
@@ -3,12 +3,7 @@ ARG TARGET=server
 # Can be used in case a proxy is necessary
 ARG GOPROXY
 
-# Build Temporal binaries
-FROM golang:1.16-alpine AS builder
-
-RUN apk add --update --no-cache \
-    make \
-    git
+FROM temporalio/base-builder AS temporal-builder
```

Let's add version tag here:
```suggestion
FROM temporalio/base-builder:1.0.0 AS temporal-builder
```

---

Same for other base images.

---

added, updated tags and images